include/networktables/NetworkTable.h

```diff
@@ -295,6 +303,15 @@ class NetworkTable : public ITable {
    * @return False if the table key already exists with a different type
    */
   virtual bool PutString(llvm::StringRef key, llvm::StringRef value) override;
+  
+  /** 
+   * Gets the current value in the table, setting it if it does not exist.
+   * @param key the key
+   * @param defaultValue the default value to set if key doens't exist.
+   * @returns the key in the table, or the defaultValue if it doesn't exist.
+   */
+  std::string SetDefaultString(llvm::StringRef key, 
+                                   llvm::StringRef defaultValue);
```

Comment:
So on some I can do that, but on some the indents like that go over 80 characters wide. Whats the best solution for methods with arguments that physically cant fit in 80 characters? For instance SetDefaultStringArray, and most of the auto val = nt::SetDefaultEntryValue functions.